controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentTester.java

```diff
@@ -69,6 +69,8 @@ public DeploymentTester(ControllerTester tester) {
 
     public ApplicationController applications() { return tester.controller().applications(); }
 
+    // TODO: This thing simulates the wrong thing: the build system won't hold the jobs that are running,
+    // and so these should be consumed immediately upon triggering, and be "somewhere else" while running.
```

Comment:
That's not what happens in the real system though. After triggering, jobs are consumed at some point in the future.

Reply:
It's not a problem right now, but it could become one, depending on how the code develops.